data/json/items/comestibles/mutagen.json

```diff
@@ -21,182 +21,83 @@
     },
     {
         "id" : "mutagen_beast",
+        "copy-from" : "mutagen",
         "type" : "COMESTIBLE",
-        "comestible_type" : "DRINK",
         "name" : "beast mutagen",
         "description" : "A rare substance of uncertain origins.  Causes you to mutate.",
```

Comment:
They probably should be however leaving redundant lines in-situ implies the descriptions differ whereas they actually don't which gives someone skimming the file the wrong impression